polyhook2/PolyHookOs.hpp

```diff
@@ -70,6 +70,15 @@
 #include <inttypes.h>
 
 void PolyHook2DebugBreak();
-void* PolyHook2Alloca(size_t size);
+
+// Methods using stack allocation need to be inlined
+__forceinline void* PolyHook2Alloca(size_t size)
```

Comment:
Can you add a new macro definition called PLH_INLINE and have it use either forceinline on msvc or otherwise __attribute__((always_inline)) for gcc and clang. That's probably enough support for now